iac-extensions/arm/src/main/java/org/sonar/iac/arm/tree/impl/bicep/ResourceDeclarationImpl.java

```diff
@@ -145,8 +147,15 @@ public StringLiteral type() {
   public List<Property> properties() {
     if (expression.is(Kind.OBJECT_EXPRESSION)) {
       return ((ObjectExpression) expression).properties();
-    } else {
+    } else if (expression.is(Kind.IF_EXPRESSION)) {
```

Comment:
Why do we call it Expression not Condition?

Reply:
Good point, I guess it's a mistake, [SONARIAC-1007](https://sonarsource.atlassian.net/browse/SONARIAC-1007) on the way to fix this.